bin/release

```diff
@@ -0,0 +1,116 @@
+  RC_BRANCH = "stages/rc-#{RELEASE_DATE}"
+
+  DRY_RUN = %w[--dry-run -n].include?(ARGV.first)
+  ARGV.clear
+
+  puts "==> This script will perform all of the following:"
+  puts "1. Checkout a new RC branch based on stages/int"
+  puts "2. Merge master into the RC branch"
+  puts "3. Push the RC branch to stages/int on GitHub"
+  puts "4. Publish the wiki page for the RC"
+
+  print "Are you sure you want to proceed? (Type 'yes' to proceed, or anything else to abort) "
+  answer = gets
+  abort "==> You have chosen to abort the script." unless answer == "yes\n"
+
+  puts "==> Checking out master and pulling latest changes"
+  run "git checkout master && git pull" unless DRY_RUN == true
+
+  if `git branch`.include?('stages/int')
+    puts "==> Checking out stages/int and pulling latest changes"
+    run "git checkout stages/int && git pull" unless DRY_RUN == true
+  else
+    puts "==> Checking out new stages/int branch and pulling down origin/stages/int"
+    run "git checkout -b stages/int origin/stages/int" unless DRY_RUN == true
+  end
+
+  unless `git branch`.include?(RC_BRANCH)
+    puts "==> Checking out RC branch for release to prod 3 days from today"
+    run "git checkout -b #{RC_BRANCH}" unless DRY_RUN == true
+
+    puts "==> Merging master into RC branch"
+    run "git merge master" unless DRY_RUN == true
+
+    puts "==> Pushing RC branch to GitHub"
+    run "git push origin #{RC_BRANCH}" unless DRY_RUN == true
+
+    # puts "===> Merge RC branch into origin/stages/int"
+    # run "git push origin #{RC_BRANCH}:stages/int" unless DRY_RUN == true
+  end
+end
+
+Dir.chdir ENV['HOME'] do
+  if `ls -d */`.include?('login-dot-gov')
+    puts "==> Pulling latest from identity-devops"
+    unless DRY_RUN == true
+      Dir.chdir("#{ENV['HOME']}/login-dot-gov/identity-devops") { run "git pull" }
+    end
+
+    puts "==> Pulling latest from identity-private.wiki"
+    unless DRY_RUN == true
+      Dir.chdir("#{ENV['HOME']}/login-dot-gov/identity-private.wiki") { run "git pull" }
+    end
+  else
+    puts "==> Creating ~/login-dot-gov directory"
+    run "mkdir login-dot-gov" unless DRY_RUN == true
+
+    puts "==> Cloning identity-devops repo into ~/login-dot-gov/identity-devops"
+    unless DRY_RUN == true
+      Dir.chdir "#{ENV['HOME']}/login-dot-gov" do
+        run "git clone https://github.com/18F/identity-devops.git"
+      end
+    end
+
+    puts "==> Cloning identity-private.wiki repo into ~/login-dot-gov/identity-private.wiki"
+    unless DRY_RUN == true
+      Dir.chdir "#{ENV['HOME']}/login-dot-gov" do
+        run "git clone https://github.com/18F/identity-private.wiki.git"
+      end
+    end
+  end
+end
+
+Dir.chdir "#{ENV['HOME']}/login-dot-gov/identity-private.wiki" do
+  rc_commit = `git log --grep="#{RC_BRANCH}" --oneline`
+  unless rc_commit.include?(RC_BRANCH)
+    puts "==> Generating the wiki page for the RC"
+    run "./scripts/generate-release --release-date=#{RELEASE_DATE}" unless DRY_RUN == true
```

Comment:
Correct. That was more complicated to do as we would have to parse the page for the previous RC and insert the new link after it. This would also require that we know what the previous RC is, which we can't easily get because we are using different naming conventions for the RC branch and the number of the RC in the Wiki.

Reply:
Or maybe there's a way to parse the Markdown and tell it to put it at the last line within the `Release candidates` bullet point.

Reply:
👍